docs/proposals/20191106-machine-load-balancer.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Machine Load Balancer 
+authors:
+  - "@moshloop"
+reviewers:
+    - "@akutz"
+    - "@ncdc"
+    - "@timothysc"
+    - "@vincepri"
+creation-date: 2019-11-06
+last-updated: 2019-11-06
+status: provisional
+---
+
+# Machine Load Balancer
+
+### Motivation
+
+* On-Premises clusters (without access to a Public Cloud Load Balancer as a Service) need a pluggable mechanism to enable the deployment of highly-available (HA) control planes using Cluster API (CAPI).
```

Comment:
Hi @moshloop,

Perhaps (_without access to a Public Cloud Load Balancer as a Service or some existing, external endpoint_)?